### DISCUSSION TITLE
PODs are not visible in ArgoCD UI 2.3.4

### DISCUSSION THREAD

**Discussion by slavhate** [argoproj/argo-cd#9542] [Q&A] [answered]
Hello there,

We recently upgraded our ArgoCD from v2.2.5 to v2.3.4. A strange issue we observe is PODs are not visible in the UI. The application tree view ends at 'Deployment' kind and it does not show the PODs after that although PODs exist in the cluster. Do we have any variable/switch that needs to be enabled/overridden to allow this view in the new version? Any specific place we need to dig down for logs related to UI?

**Comment by slavhate** [comment, marked as answer]
For anyone who is referring to this thread, we got this resolved by :
Adding ReplicaSet and Pod into the ArgoCD Projects configuration in the below section -
```bash
  namespaceResourceWhitelist:
    - group: ""
      kind: Pod
    - group: apps
      kind: ReplicaSet`
```
A shout out to the #argo-cd Slack channel where user [jannfis](https://app.slack.com/team/U01G13HQNQ3) helped us with this solution!

Referneces:
https://github.com/argoproj/argo-cd/releases/tag/v2.3.2
https://github.com/argoproj/argo-cd/security/advisories/GHSA-2f5v-8r3f-8pww

**Reply by mikecarr** [reply]
Do you have to reploy all apps afterwards? I just aadded this yet Argo still does not reflect thge change
